.gitignore

```diff
@@ -3,7 +3,7 @@
 /Manifest.toml
 /deps/build
 /deps/build.log
-/deps/deps-*.jl
+/deps/deps*.jl
```

Comment:
Hmm, any `deps/deps.jl` file you have should be a left-over from an older version, and would best be removed. This is why I didn't make the ignore rule so broad. Do you have a specific reason you need it like this?

Reply:
@fingolfin I have a `deps/deps.jl` from May 5th, which was last changed a minute before `deps-1.4.jl` and `build.log` were changed for the last time.